src/CoreShop/Bundle/FrontendBundle/Resources/config/services.yml

```diff
@@ -2,6 +2,11 @@ imports:
     - { resource: services/wishlist.yml }
     - { resource: services_alias.yml }
 
+parameters:
+    coreshop.frontend.category.valid_sort_options: ['name']
```

Comment:
can we use bundle configuration for that?

```
core_shop_frontend:
    category:
        valid_sort_options: ['name']
        default_sort_name: 'name'
        default_sort_direction: 'asc' #only allow asc, desc
```

Reply:
Changes made, please check it now.